src/object/cli_obj.c

```diff
@@ -4231,6 +4336,21 @@ obj_comp_cb(tse_task_t *task, void *data)
 		obj_auxi->ec_in_recov = 0;
 		obj_reasb_io_fini(obj_auxi, true);
 		D_DEBUG(DB_IO, DF_OID" EC fetch again.\n", DP_OID(obj->cob_md.omd_id));
+	} else if (obj_req_is_ec_cond_fetch(obj_auxi) && task->dt_result == -DER_NONEXIST &&
+		   !obj_auxi->ec_degrade_fetch && !obj_auxi->cond_fetch_split) {
```

Comment:
Consider L4258, the "task->dt_result" should has been covered as "0" under such case, right? So can we arrive at this branch?

Reply:
there are different handling, L4258 is for the case that "obj_auxi->result == 0 && task->dt_result == -DER_NONEXIST", it means some shard get 0 (exist) and some shards get NONEXIST - so should zero the task->dt_result.
here is the case that all shards got NON-EXIST.